example/lib/flap/flap_demo.dart

```diff
@@ -14,8 +14,8 @@ class FlapDemo extends StatelessWidget {
       valueListenable: themeNotifier,
       builder: (_, ThemeMode currentMode, __) {
         return MaterialApp(
-          theme: adwaita.lightTheme,
-          darkTheme: adwaita.darkTheme,
+          theme: adwaita.AdwaitaThemeData.light(),
```

Comment:
With adwaita 0.0.4, there is no need for the namespace `adwaita`. Could you remove it :)?